js/siteorigin-panels/view/live-editor.js

```diff
@@ -19,7 +20,7 @@ module.exports = Backbone.View.extend( {
      * Render the live editor
      */
     render: function(){
-        this.$el.html( this.template() );
+        this.setElement( this.template() );
```

Comment:
Aaah yeah, forgot about the `return this;`. This view should only be getting rendered once because it's not associated with a model, but from a good-practices point of view, I'd be happy to change. So far all `render` functions in PB are similar to this, so would be a pretty big change for all of them. Will look into it.